sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md

```diff
@@ -0,0 +1,67 @@
+# Recognize form content
+
+This sample demonstrates how to recognize tables, lines, and words in documents, without the need to train a model.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `FormRecognizerClient`
+
+To create a new `FormRecognizerClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if neded, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormRecognizerClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormRecognizerClient(new Uri(endpoint), credential);
+```
+
+## Recognize form content from an Uri
```

Comment:
```suggestion
## Recognize form content from a Uri
```

Reply:
ohh really? do you mind explaining why?

Reply:
I would do this because we pronounce Uri as "You-R-I" ... so the "a" comes before a consonant _sound_, even though the word starts with a vowel.

Reply:
Thank you!